README.md

```diff
@@ -17,6 +17,13 @@ This repo includes the following components:
 
 ## Project Setup
 
+After you've succesfully cloned OpsCenter reprository, please run:
+
+```
+git submodule init
```

Comment:
I suggest we do a clone that is recursive as opposed to these submodule commands. 

e.g. `git clone --recurse-submodules ...`